metasmoke.py

```diff
@@ -56,6 +57,7 @@ def connect_websocket():
     @staticmethod
     def init_websocket():
         has_succeeded = False
+        failed_connection_attempts = 0
         while GlobalVars.metasmoke_key != "":
```

Comment:
If we're going to retry, then `while` should probably be:

    while GlobalVars.metasmoke_key != "" and GlobalVars.metasmoke_ws_host !="":

because a user may have modified the config file to remove either of those values thinking that it will disable the MS WebSocket.